Documentation.md

```diff
@@ -1036,7 +1038,8 @@ agent is refreshed.
 
 ## Enumerations
```

Comment:
Added some missing `enum`s mentioned in the document, and also completed incomplete enumerations.